src/lib/shaders/scene-vertex-shader.ts

```diff
@@ -297,15 +302,20 @@ void main () {
         targetBorderRadiusPixel(),
         targetBorderPlacement())
   );
+  float currentBorderRadiusWorld = borderProperties.x;
+  float currentBorderRadiusPixel = borderProperties.y;
+  float currentBorderPlacement = borderProperties.z;
 
   // The fragment shader needs to know the threshold signed distances that
   // indicate whether each pixel is inside the shape, in the boreder, or outside
   // of the shape.
   vec2 projectedSizePixel = computedSize.xy * viewMatrixScale.xy;
-  float edgeDistance = borderProperties.x +
-    borderProperties.y * 8. / min(projectedSizePixel.x, projectedSizePixel.y);
+  float edgeDistance = currentBorderRadiusWorld + (
+      currentBorderRadiusPixel * 4. * devicePixelRatio /
```

Comment:
The multiplication by 4 here seems like a bit of a magic number to me. Worth documenting?

Reply:
Thanks for the review. It's definitely a magic number. Let me see if I can figure out what it means.

Reply:
Explanation added, PTAL.

Reply:
Thanks, those two constants make the whole computation a lot more obvious!

Reply:
No worries. Thanks for bringing these issues up. There was a lot of development in the beginning wherein I was focused on making it look right (the rendered output, not the code). Some of these magic values are holdovers from that time.